pkg/apis/extensions/validation/validation.go

```diff
@@ -656,6 +656,25 @@ func ValidateScale(scale *extensions.Scale) field.ErrorList {
 		allErrs = append(allErrs, field.Invalid(field.NewPath("spec", "replicas"), scale.Spec.Replicas, "must be greater than or equal to 0"))
 	}
 
+	if scale.Status.Selector == nil {
+		return append(allErrs, field.Required(field.NewPath("status", "selector"), ""))
+	}
+
+	return allErrs
+}
+
+func ValidateScaleUpdate(scale *extensions.Scale) field.ErrorList {
+	allErrs := field.ErrorList{}
+	allErrs = append(allErrs, apivalidation.ValidateObjectMeta(&scale.ObjectMeta, true, apivalidation.NameIsDNSSubdomain, field.NewPath("metadata"))...)
```

Comment:
I thought @DirectXMan12 tried adding scale update validation and it was decided it was unnecessary because the scale object's metadata was synthetic (no creation time stamp, etc)